commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AbstractAuditLogSyslogReliableProtocolCommand.java

```diff
@@ -0,0 +1,105 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Batch;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+abstract class AbstractAuditLogSyslogReliableProtocolCommand extends AbstractAuditLogSyslogHandlerCommand {
```

Comment:
Oooops, I forgot to remove both apply methods from this class. They definitely shouldn't be there.

Anyway, did I get your point right that you suggest creation of some util class that would cover the common functionality of _apply_ methods in _AddAuditLogTcpSyslogHandler_ and _AddAuditLogTlsSyslogHandler_ commands rather than extending _AbstractAuditLogSyslogReliableProtocolCommand_ (resulting in removal of this class)?